projects/go-tester/src/app/app.component.scss

```diff
@@ -1 +1 @@
-
+@import "~@ng-select/ng-select/themes/default.theme.css";
```

Comment:
I know this is for our tester app, but do we need to import this in the project separately? It looks like it's being imported in the component's SCSS file as well, are those styles not applying?

Reply:
This was a leftover... removed it